app/[games]/page.tsx

```diff
@@ -0,0 +1,10 @@
+import EditGameForm from "@/components/EditGameForm";
+
+export const metadata = {
+  title: "Edit game",
+  description: "Here you can edit game that you added",
+};
+
+export default function EditPage({ params }: { params: { games: number } }) {
```

Comment:
`games` param name is a bit off, it's just a bit miss leading as it is actually a single game id. So how about just `id` instead?